Games/JV.AD/script.js

```diff
@@ -0,0 +1,73 @@
+//making timer 
+let timeLeft = 60;
+let timerInterval;
+function startGame()
+  {
+    document.getElementById("startBtn").disabled = true;
+    
+    nextQuestion();
+
+    let timeDisplay=document.getElementById("timeDisplay");
+    timeDisplay.hidden = false;
+    timerInterval = setInterval(function(){
+      timeLeft -= 1;
+      
+      timeDisplay.innerHTML = "Time Left: " + timeLeft;
+      if (timeLeft == 0)
+      {
+        clearInterval(timerInterval);
+      }
+    },1000)
+  }
+
+
+//making randomizer with sin,cos, and tan to put in questions 
+var trigratios = ["sin","cos","tan"];
+var trigratio =  trigratios[Math.floor(Math.random()*trigratios.length)];
+
+//creating function to be able to change trig ratios in question 
+function nextQuestion()
+{
+question = document.getElementById("question");
+question.innerHTML = "What is the trig ratio of " + trigratio + "?";
```

Comment:
Use [string interpolation](https://dmitripavlutin.com/string-interpolation-in-javascript/) rather than string concatenation for better readbility and (in the future) internationalization (translations).